app/src/main/res/values/strings.xml

```diff
@@ -556,17 +556,26 @@ be temporary, and you can try again later.</li>
     <string name="menu_drawer_home">Home</string>
 
     <!-- Home tile strings -->
+
+    <!-- Trademarked brand name: Does not need to be translated -->
```

Comment:
AH! These should get removed by #440, so up to you if we should wait on that or if you want to land this now.